src/collections/_documentation/common/getting_started/getting_started_template.md

```diff
@@ -0,0 +1,124 @@
+<!--
+Guideline: This page is common to all SDKs; add the name of the SDK you're documenting as well as the code samples as instructed. If you have questions, please ask Fiona or Daniel. 
+
+**The objective for this page is that a developer can quickly complete an ideal integration.**
+-->
+
+Add Sentry's <!--SDK name--> SDK to automatically report errors, exceptions, and rejections in your application. 
+
+On this page, we provide concise information to get you up and running quickly. To learn more, use the links to in-depth content.
+
+In the right place? We also offer documentation for:
+<!-- 
+Guideline: Add icons for related languages/frameworks.
+-->
+What's covered on this page:
+
+- [Install](#install)
+- [Initialize](#initialize)
+- [Verify Setup](#verify-setup)
+- [Capture Errors](#capture-errors)
+    1. [Enrich Error Data](#enrich-error-data) 
+    2. [Set the Release Version](#set-release-version)
+    
+<!--
+Guideline: Add any step to set up information that's specific to the SDK; for example, in the JavaScript example, this heading is for Source Maps. Then add the link to that heading
+-->
+
+- [Monitor Performance](#monitor-performance)
+
+## Install {#install}
+
+1.  If you already have an account, log in to Sentry. If not, establish your account:
```

Comment:
i strongly would urge us to get this out of docs. if you don't know how to sign-up for a sentry account then you shouldn't be here in the first place. to me this is marketing material not documentation. almost _always_ someones going to end up here from within the application (they need 'more' stuff), or because they know how to configure sentry and are looking for the language-specific guide.

Reply:
We can change it simply to "If you don't have an account, got to Sentry.io and set one up, then come back here"?

Reply:
why do we need to tell them that at all? Just tell them where they can get their DSN and call it a day imo. we have to expect they have some level of knowledge, and that they're not here to be told how to create an account (or that they need one to use sentry), right?

Reply:
I've modified to assume an account, or, if not, set one up. LMK if this draft works.